common/lib/xmodule/xmodule/split_test_module.py

```diff
@@ -158,7 +161,23 @@ class SplitTestBlock(
 
     show_in_read_only_mode = True
 
+    preview_view_js = {
+        'js': [],
+        'xmodule_js': resource_string(__name__, 'js/src/xmodule.js'),
+    }
+    preview_view_css = {
+        'scss': [],
+    }
+
     mako_template = "widgets/metadata-only-edit.html"
+    studio_js_module_name = 'SequenceDescriptor'
+    studio_view_js = {
+        'js': [resource_string(__name__, 'js/src/sequence/edit.js')],
+        'xmodule_js': resource_string(__name__, 'js/src/xmodule.js'),
+    }
+    studio_view_css = {
+        'scss': [],
+    }
```

Comment:
SequenceBlock does not need an edit view so we move the references to the files here so they get compiled into `SplitTestBlockStudio`.

Reply:
Question: This definitely doesn't break Studio's edit interface? I ask only because showing a sequence-like nav at the top of the unit editor in Studio is a semi-recent addition, and I don't remember how it was implemented.

Reply:
I do not recall anything broken while testing. But I can take a look at any implications of this change again.

Reply:
It doesn't use `studio_view()` but the `author_view()` of `SequenceBlock` and if we don't update the static assets results in this error:

```
  File "/edx/app/edxapp/edx-platform/common/lib/xmodule/xmodule/seq_module.py", line 451, in author_view
    return self._student_or_public_view(context, True, {}, view=AUTHOR_VIEW)
  File "/edx/app/edxapp/edx-platform/common/lib/xmodule/xmodule/seq_module.py", line 556, in _student_or_public_view
    add_webpack_to_fragment(fragment, 'SequenceBlockPreview')
  File "/edx/app/edxapp/edx-platform/common/lib/xmodule/xmodule/util/xmodule_django.py", line 29, in add_webpack_to_fragment
    for chunk in webpack_loader.utils.get_files(bundle_name, extension, config):
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/webpack_loader/utils.py", line 48, in get_files
    return list(_get_bundle(bundle_name, extension, config))
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/webpack_loader/utils.py", line 40, in _get_bundle
    bundle = get_loader(config).get_bundle(bundle_name)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/webpack_loader/loader.py", line 82, in get_bundle
    raise WebpackBundleLookupError('Cannot resolve bundle {0}.'.format(bundle_name))
webpack_loader.exceptions.WebpackBundleLookupError: Cannot resolve bundle SequenceBlockPreview.
```

![image](https://user-images.githubusercontent.com/5305552/109332665-3d07cb00-7880-11eb-9ce3-516785e40652.png)

But that set up has been updated so it works as it does in LMS.

![image](https://user-images.githubusercontent.com/5305552/109331547-c918f300-787e-11eb-8289-7b9c00f98859.png)